hera_cal/redcal.py

```diff
@@ -147,6 +150,71 @@ def get_reds(antpos, pols=['xx'], pol_mode='1pol', ex_ants=[], bl_error_tol=1.0,
     return add_pol_reds(pos_reds, pols=pols, pol_mode=pol_mode, ex_ants=ex_ants)
 
 
+def filter_reds(reds, bls=None, ex_bls=None, ants=None, ex_ants=None, ubls=None, ex_ubls=None, pols=None, ex_pols=None):
+    '''
+    Filter redundancies to include/exclude the specified bls, antennas, unique bl groups and polarizations.
+    Assumes reds indices are Antpol objects.
+    Args:
+        reds: list of lists of redundant baseline tuples, e.g. (ind1,ind2,pol)
+        bls (optional): baselines as antenna pair tuples (i,j) to include in reds.
```

Comment:
You should clarify that this supports (i,j) and (i,j,pol) formats.

Reply:
Done.